source/docs/steeltoe-configuration.html.markdown

```diff
@@ -404,9 +440,9 @@ See [Push Sample](#push-sample) for instructions on how to push this sample to e
 
 #### 2.1.2.4 Observe Logs
 
-To see the logs as you startup the application use the `cf` CLI to tail the apps logs. (i.e. `cf logs foo`)
+To see the logs as you start the application, use `cf logs sample`.
```

Comment:
I believe the app name is 'foo' in the sample, we either need to change the docs or the sample. 
@TimHess What do you think?

Reply:
https://github.com/spring-cloud-samples/config-repo has 'foo' in it.